src/CodeStyle/Core/CodeFixes/FormattingCodeFixProvider.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, LanguageNames.VisualBasic, Name = "FixFormatting")]
+    [Shared]
+    internal class FormattingCodeFixProvider : CodeFixProvider
```

Comment:
>FormattingCodeFixProvider [](start = 19, length = 25)

Is it possible we can link this file/fixer into the Features layer instead of defining an identical SyntaxEditorBasedCodeFixProvider there?

Reply:
I was able to share `FixOneAsync`.